docs/explanations/data_collection.rst

```diff
@@ -0,0 +1,86 @@
+What data does Canonical collect from Ubuntu Pro machines?
+**********************************************************
+
+For the purposes of delivering Ubuntu Pro services to customers in compliance
+with the terms of the Ubuntu Pro subscription, some system data is sent to
+Canonical servers. This data is sent via a couple different methods, depending
```

Comment:
I'd flip the order of this around to make what we're doing explicitly stated at the start, i.e.:
"Some system data is sent to Canonical servers for the purpose of delivering Ubuntu Pro services in compliance with the terms of the Ubuntu Pro subscription."